src/fiatconnect/ReviewScreen.tsx

```diff
@@ -334,7 +339,6 @@ function getDisplayAmounts({
   networkFee: BigNumber
   tokenInfo: TokenBalance | undefined
 }) {
-  const cryptoType = normalizedQuote.getCryptoType()
```

Comment:
Golf `const tokenId = normalizedQuote.getTokenId()` similar to what was done with `cryptoType` before